audit/audit.cc

```diff
@@ -121,7 +121,7 @@ audit::audit(locator::shared_token_metadata& token_metadata,
 
 audit::~audit() = default;
 
-future<> audit::create_audit(const db::config& cfg, sharded<locator::shared_token_metadata>& stm) {
+future<> audit::start_audit(const db::config& cfg, sharded<locator::shared_token_metadata>& stm, sharded<cql3::query_processor>& qp, sharded<service::migration_manager>& mm) {
```

Comment:
long term audit::start_audit and audit::start (same with stop and shutdown) should be merged too, but that's good incremental step